data/telegraf_plugins.yml

```diff
@@ -1455,6 +1493,15 @@ input:
     introduced: 1.7.0
     tags: [linux, macos, systems]
 
+  - name: Synproxy
+    id: synproxy
+    description: |
+      The Synproxy plugin gathers the synproxy counters.
```

Comment:
Maybe

`The Synproxy plugin gathers synproxy information`
or
`The Synproxy plugin gathers ~the~ synproxy counters.`